1-the-basics/index.html

```diff
@@ -12,8 +12,11 @@ <h3>Hobbies</h3>
     <ul>
         <li>Read</li>
         <li>Watch movies<li>
+        <li>Second<li>
+        <li>Second...<li>
+        <li>Four<li>
     </ul>
-    <p>I'm a  <strong></strong>grumy</p>
+    <p>My cat is <strong>very</strong>grumy</p>
     <img height="400" width="500" src="https://s3.amazonaws.com/rtvc-assets-senalcolombia.gov.co/s3fs-public/field/image/mi-vecino-totoro-mei-satsuke-lluvia-parada-bus.jpg"alt="My new image ">
     <br>
```

Comment:
Only one more comment before merging with master, I forgot it. Using `<br>` it's a bad practice because it's not a semantic tag. We are going to see how to put a line break with CSS. And remove the empty lines please.